Assignment6/RearrangingCars.py

```diff
@@ -0,0 +1,71 @@
+class Parking:
+
+    def __init__(self, cars):
+        self._cars = cars
+        self._size = len(cars)
+
+    def cars(self):
+        return self._cars.copy()
+
+    def size(self):
+        return self._size
+
+    def rearrange(self, result):
+        """
+        Rearrange cars in self._cars to get the result and print out each step.
+        :param result: list(int)
+        :return: None or raise ValueError
+        """

```

Comment:
Thanks! It is automatically created comments in PyCharm editor. It is very comfortable. 
And it is good idea to add some extra information, but do you think it is better to add it near types or it is good in the description as well? (it will be more beautiful :)